protocols/developer_dao/index.json

```diff
@@ -0,0 +1,11 @@
+{ 
+	"cname": "developer_dao",
```

Comment:
Hey ! Could you change cname and path to developerdao please? Without the underscore.

Reply:
Done!